reasoner/controller/ConditionController.java

```diff
@@ -45,17 +43,12 @@ public class ConditionController extends ConjunctionController<
         this.condition = condition;
     }
 
-    @Override
-    Set<Concludable> concludablesTriggeringRules() {
```

Comment:
No longer needed, since these can be answered by the conjunction in all subclasses of ConjunctionController.

Reply:
Outdated; Was removed in a different PR